app/controllers/Api.scala

```diff
@@ -123,6 +123,20 @@ class Api (cc: ControllerComponents, prismDataStore: Prism, prismConfiguration:
       Api.singleItem(prismDataStore.instanceAgent, arn)
     }
 
+    def instanceAccounts: Action[AnyContent] = Action.async { implicit request =>
+      ApiResult.noSource {
+        val accounts = prismDataStore.instanceAgent.get().map { c =>
```

Comment:
I don't think this should hang off the `instanceAgent`. There could be an account that we don't crawl instances for.

Reply:
See sources on line 74.

Reply:
I think the diff might not have updated - this now uses the sources agent

Reply:
So it does. Weird.